docs/.vuepress/components/guide/readme/dp-disabled-dates.vue

```diff
@@ -2,18 +2,34 @@
   <div class="example">
     <v-date-picker
       v-model="date"
-      :disabled-dates="{ weekdays: [1, 7] }"
-      is-range
+      :disabled-dates="disabled"
+      :columns='2'
+      timezone='Pacific/Auckland'
+      @update:to-page='onUpdateToPage'
```

Comment:
`timezone` and `disabled-dates` props are required to reproduce the bottleneck.

The to-page handler is just to get an approximate handle on when pages finish to end the timer.

I think `columns` prop exaggerates the severity of the bottleneck.